test/exec.js

```diff
@@ -35,7 +35,7 @@ test('config.fatal and unknown command', t => {
   shell.config.fatal = true;
   t.throws(() => {
     shell.exec('asdfasdf'); // could not find command
-  }, /exec: internal error/);
+  }, /((asdfasdf: not found)|(asdfasdf: command not found)|('asdfasdf' is not recognized))/);
```

Comment:
Why 3 regexes? If these are different shells, let's put a comment.

Should we check it's a shelljsinternalerror?